third_party/blink/renderer/modules/cobalt/h5vcc_runtime/h_5_vcc_runtime.h

```diff
@@ -54,6 +54,14 @@ class MODULES_EXPORT H5vccRuntime final
   EventListener* ondeeplink();
   void setOndeeplink(EventListener* listener);
 
+  // EventTarget interface:
+  bool AddEventListenerInternal(
```

Comment:
It seems like the sibling method `AddedEventListener()` (not `Add...`) is much more commonly overridden (clicking on it in e.g. https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/dom/events/event_target.h;l=342;drc=260b80710982c3d47bddfe6b075a80a0ad6cd84a;bpv=1;bpt=1) than this -- also it's odd to overwrite something called "Internal". I'm just double checking this is the intended functionality?